src/site/xdoc/server/config-blob.xml

```diff
@@ -0,0 +1,109 @@
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobId.java">BlobId</a>.
+              Clients can use BlobStore to retrieve stored data by the BlobId.
+              <dl>
+                  <dt><strong>implementation</strong></dt>
+                  <dd>
+                      Define the BlobStore implementation will be used in the product. Available implementations: cassandra, objectstorage, union.
+                      cassandra: blobs will be stored and retrieved at cassandra database.
+                      objectstorage: blobs will be stored and retrieved at open-stack swift.
+                      union: blobs will be stored and retrieved at the current BlobStore(open-stack swift), if fails or not found, it fallback to use the legacy BlobStore (cassandra)
+                      Default is cassandra.
+                  </dd>
+              </dl>
+          </p>
+
+          <subsection name="Swift BlobStore Configuration">
+              <dl>
+                  <dt><strong>objectstorage.payload.codec</strong></dt>
+                  <dd>AES256: Aes256 encrypted blob content. DEFAULT: non encrypted</dd>
+
+                  <dt><strong>objectstorage.provider</strong></dt>
+                  <dd>Should be swift</dd>
+
+                  <dt><strong>objectstorage.namespace</strong></dt>
+                  <dd>Namespace to store blobs in</dd>
+
+                  <dt><strong>objectstorage.swift.authapi</strong></dt>
+                  <dd>Should be tmpauth</dd>
+
+                  <dt><strong>objectstorage.swift.endpoint</strong></dt>
+                  <dd>Swift authentication endpoint. Should be http://swift-host-address:8080/auth/v1.0 </dd>
+
+                  <dt><strong>objectstorage.swift.credentials</strong></dt>
+                  <dd>Should be testing</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.username</strong></dt>
+                  <dd>Should be tester</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.tenantname</strong></dt>
+                  <dd>Should be test</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
+                  <dd>Should be X-Storage-Pass</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
+                  <dd>Should be X-Storage-User</dd>
+              </dl>
+          </subsection>
+      </subsection>
+
+      <subsection name="Blob Export Configuration">
+          <p>
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-export-api/src/main/java/org/apache/james/blob/export/api/BlobExportMechanism.java">BlobExportMechanism</a>
+              is a component used to export blob content to a target email address. It requires to has a BlobId, an string of explanation, and target mail address in order to perform exporting.
```

Comment:
It requires to `s/has/have` a BlobId, `s/an/a` string of explanation, and `s//a` target mail address in order to perform `s/exporting/exports`.